releasenotes/notes/add-ddtrace-deprecation-category-f435cd2447d97899.yaml

```diff
@@ -0,0 +1,16 @@
+---
+prelude: >
+  In preparation for the upcoming v1.0.0 release, we recommend upgrade to ``ddtrace>=0.59.0,<1.0.0`` and configuring `warning filters <https://docs.python.org/3.8/library/warnings.html#warning-filter>`_.
+
+  Users of pytests can add a filter for ``ddtrace.DDTraceDeprecationWarning``::
+    pytest -W "error::ddtrace.DDTraceDeprecationWarning" tests.py
+
+
+  Otherwise, set the environment variable ``DD_TRACE_RAISE_DEPRECATIONWARNING``::
```

Comment:
Should we explicitly state "to turn deprecation warnings into exceptions"?

Wonder if this is clear enough that your app will crash with this setting enabled?